assets/wizards/connections/views/main/google.js

```diff
@@ -39,18 +39,18 @@ const GoogleOAuth = ( { setError, onInit, onSuccess } ) => {
 	useEffect( () => {
 		if ( isConnected && ! userBasicInfo.has_refresh_token ) {
 			setError( [
-				__( 'Missing Google refresh token. Please', 'newspack' ),
+				__( 'Missing Google refresh token. Please', 'newspack-plugin' ),
 				' ',
 				<a
 					key="link"
 					target="_blank"
 					rel="noreferrer"
 					href="https://myaccount.google.com/permissions"
 				>
-					{ __( 'revoke credentials', 'newspack' ) }
+					{ __( 'revoke credentials', 'newspack-plugin' ) }
 				</a>,
 				' ',
-				__( 'and authorise the site again.', 'newspack' ),
+				__( 'and authorise the site again.', 'newspack-plugin' ),
```

Comment:
```suggestion
				__( 'and authorize the site again.', 'newspack-plugin' ),
```

An opportunity to :us:-ize the spelling here, for consistency. 😄